tests/test_playbooks/repository_ostree.yml

```diff
@@ -40,7 +40,7 @@
     - name: sync repository
       include_tasks: tasks/katello_sync.yml
       vars:
-        repository: "Test Ostree Repository"
+        repository: "Test OStree Repository"
```

Comment:
So this typo triggered the bug and we never noticed?
Should we keep a version of this expecting to properly fail now?

Reply:
Correct, the typo triggered the wrong codepath being executed.

I'll merge as-is and add a test in a separate PR.

Reply:
https://github.com/theforeman/foreman-ansible-modules/pull/1532